src/utils.js

```diff
@@ -195,9 +195,15 @@ export async function* withPage(percy, url, callback, retry, args) {
       attempt++;
       let enableRetry = process.env.PERCY_RETRY_STORY_ON_ERROR || 'true';
       if (!(enableRetry === 'true') || attempt === retries) {
+        // Add snapshotName to the error message
+        const snapshotName = args?.snapshotName;
+        if (snapshotName) {
+          error.message = `${error.message} - Snapshot Name: ${snapshotName}`;
```

Comment:
Shouldnt this be Snapshot name followed by error.message ? wouldnt that make it easy to understand snapshot in case of long error messages ?

Reply:
Also the original comment is not handled. THe message should be passed from outside. Considering "Retrying Story" makes no sense if we are opening preview page

Reply:
Passed form inside try when attempt number > 0.